Added EA docs for Slack personal and team notifications (#3900)
getsentry/sentry-docs: src/docs/product/integrations/notification-incidents/slack/index.mdx

## Configure

Use Slack for notifications and [alerts](#alert-rules) regarding issues, environments, deployment, etc.

### Personal Notifications

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

Let us know if you have feedback: [ecosystem-feedback@sentry.io](mailto:ecosystem-feedback@sentry.io).

</Note>

You can receive personal workflow, deploy, and issue alert notifications from our Slack integration. Manage your [personal notification settings](/product/alerts/notifications/notification-settings/) by navigating to **User Settings > Notifications**.

#### Linking Your Slack and Sentry Accounts

> In order to receive personal notifications from our Slack integration, your Slack identity must be linked with your Sentry account. This can be done in Slack by using the `/sentry link` command.

Reviewer: As a user I _might_ not know what "using the `/sentry link` command" means, what do you think about changing it to "...by typing `/sentry link` in Slack"?

### Team Notifications

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

Let us know if you have feedback: [ecosystem-feedback@sentry.io](mailto:ecosystem-feedback@sentry.io).

</Note>

> You can receive [team alert notifications](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions) from our Slack integration. To enable this feature, use the `/sentry link team` command in the desired Slack channel. To view a team's associated Slack channel in [sentry.io](https://sentry.io), navigate to **Settings > Teams > [Team] > Notifications**.

Reviewer: same as above here

## Configure

Use Slack for notifications and [alerts](#alert-rules) regarding issues, environments, deployment, etc.

### Personal Notifications

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

Let us know if you have feedback: [ecosystem-feedback@sentry.io](mailto:ecosystem-feedback@sentry.io).

</Note>

> You can receive personal workflow, deploy, and issue alert notifications from our Slack app. You can manage your [personal notification settings](/product/alerts/notifications/notification-settings/) by navigating to *User Settings > Notifications*.

Reviewer 2: Wonder if we should use "Slack integration" rather than "Slack app" to avoid confusion.

```suggestion
You can receive personal workflow, deploy, and issue alert notifications from our Slack integration. Manage your [personal notification settings](/product/alerts/notifications/notification-settings/) by navigating to **User Settings > Notifications**.
```

> #### Linking your Slack and Sentry accounts

Reviewer 2: ```suggestion
#### Linking Your Slack and Sentry Accounts
```

> In order to receive personal notifications from our Slack app, your Slack identity must be linked with your Sentry account. This can be done in Slack by using the `/sentry link` command.

Reviewer 2: Same suggestion as above re: integration vs app

### Team Notifications

<Note>

> This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
> 
> If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

Reviewer 2: ```suggestion
This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
```

Let us know if you have feedback: [ecosystem-feedback@sentry.io](mailto:ecosystem-feedback@sentry.io).

</Note>

> You can receive [team alert notifications](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions) from our Slack app. This can be done in Slack by using the `/sentry link team` command in the desired channel. To view a team's associated Slack channel in Sentry navigate to *Settings > Teams > [Team] > Notifications*.

Reviewer 2: ```suggestion
You can receive [team alert notifications](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions) from our Slack integration. To enable this feature, use the `/sentry link team` command in the desired Slack channel. To view a team's associated Slack channel in [sentry.io](https://sentry.io), navigate to **Settings > Teams > [Team] > Notifications**.
```

We recommend disabling the legacy project-based integration after setting up the global integration.

> Once you configure the global Slack integration and Alert Rules, you can disable the old project-based Slack integration. Go to each project that has the old project-based Slack enabled and disable it.

Reviewer 2: ```suggestion
Once you configure the global Slack integration and alert rules, you can disable the old project-based Slack integration. Go to each project that has the old project-based Slack enabled and disable it.
```
